pilot/pkg/networking/util/util_test.go

```diff
@@ -1227,3 +1227,269 @@ func TestStatefulSessionFilterConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestEndpointTLSModeLabel(t *testing.T) {
```

Comment:
This test does not make much sense, i meant testing on the `buildLocalityLbEndpointsFromShards` is better